src/data/complex/module.lean

```diff
@@ -79,6 +79,18 @@ instance [comm_semiring R] [algebra R ℝ] : algebra R ℂ :=
   commutes' := λ r ⟨xr, xi⟩, by ext; simp [smul_re, smul_im, algebra.commutes],
   ..complex.of_real.comp (algebra_map R ℝ) }
 
+/-- Complex conjugation as an `ℝ`-algebra isomorphism  -/
+def conj_real_alg_equiv : ℂ ≃ₐ[ℝ] ℂ :=
+{ inv_fun := complex.conj,
+  left_inv := complex.conj_conj,
+  right_inv := complex.conj_conj,
+  commutes' := complex.conj_of_real,
+  .. complex.conj }
+
+/-- Complex conjugation as an `ℚ`-algebra isomorphism  -/
+def conj_rat_alg_equiv : ℂ ≃ₐ[ℚ] ℂ :=
+alg_equiv.restrict_scalars ℚ complex.conj_real_alg_equiv
```

Comment:
I'd be tempted to remove this one, and just use `conj_real_alg_equiv.restrict_scalars ℚ` in its place. You could remove `_real` from the name to make that a bit shorter.